packages/devextreme/js/__internal/ui/chat/chat_avatar.ts

```diff
@@ -7,18 +7,23 @@ import Widget from '@ts/core/widget/widget';
 
 const CHAT_MESSAGE_AVATAR_CLASS = 'dx-chat-message-avatar';
 const CHAT_MESSAGE_AVATAR_INITIALS_CLASS = 'dx-chat-message-avatar-initials';
+const CHAT_MESSAGE_AVATAR_IMAGE_CLASS = 'dx-chat-message-avatar-image';
 
 export interface Properties extends WidgetOptions<Avatar> {
   name?: string;
+  url?: string;
 }
 
 class Avatar extends Widget<Properties> {
-  _$initials!: dxElementWrapper;
+  _$initials?: dxElementWrapper;
+
+  _$image?: dxElementWrapper;
 
   _getDefaultOptions(): Properties {
     return {
       ...super._getDefaultOptions(),
       name: '',
+      url: '',
```

Comment:
Kindly use defaultOption.tests.js to test this setting

Reply:
[4da065f777adfedddf3bf741d68b4ba898273434]